common/app/views/support/HtmlCleaner.scala

```diff
@@ -500,6 +500,16 @@ case class DropCaps(isFeature: Boolean) extends HtmlCleaner {
         case _ =>
       }
     }
+
+    document.getElementsByTag("h2").foreach{ h2 =>
+        if (h2.text() == "* * *") {
```

Comment:
It's a common thing they do on long reads. Eventually this would be done using the section headings, but want to keep it scoped while testing.